backend/src/models/Project.ts

```diff
@@ -30,6 +31,10 @@ const ProjectSchema = new Schema<IProject>(
       type: [Schema.Types.ObjectId],
       ref: "tech",
     },
+    starrers: {
+      type: [Schema.Types.ObjectId],
+      ref: "user",
```

Comment:
we should default this to empty array, otherwise, `p.starrers` is undefined in the frontend for some projects

```
starrers: {
  type: [Schema.Types.ObjectId],
  ref: "user",
  default: [], <-
}
```